### PR TITLE
Add factbook

### PR DESCRIPTION
**Pull request by tmilner**

### CODE REVIEW

**Comment by gmiller-mdsol** [comment]
Are there any teams/product owners attached to this?

**Comment by jcarres-mdsol** [comment]
Yeah, there are still a lot of things that can be added without adding personal information

File: factbook.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "https://github.com/mdsol/platform-standards/tree/master/schemas/v1alpha1.schema.json"
+  name: mauth-jvm-clients
+  description: MAuth Clients for JVM based applications.
+  teams:
+    - name: "Team 73 - Navy Seals"
+      number: 73
+    - name: Team 259
+      number: 259
+    - name: Team 25
+      number: 55
```
Comment (mayman) [inline factbook.yaml:18]: These two are inconsistent

File: factbook.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    # This is a constant, just leave as is.
+    "https://github.com/mdsol/platform-standards/tree/master/schemas/v1alpha1.schema.json"
+  name: mauth-jvm-clients
+  description: MAuth Clients for JVM based applications.
+  teams:
+    - name: "Team 73 - Navy Seals"
+      number: 73
+    - name: Team 259
+      number: 259
+    - name: Team 25
```
Comment (tmilner) [inline factbook.yaml:17]:
```suggestion
    - name: Team 55
```

**Review by fserra-mdsol — APPROVED** [review]

**Review by mayman — APPROVED** [review]

**Pull request merged**